src/components/Results/DetectedIssueResources.tsx

```diff
@@ -36,63 +51,66 @@ const DetectedIssueResources: React.FC<Props> = ({ detectedIssue }) => {
+                        {fhirpath.evaluate(response, 'dataRequirement.codeFilter.valueSet')}
+                      </Link>
                     </Typography>
                   </Grid>
                   <Grid item xs>
-                    path: {fhirpath.evaluate(response, 'dataRequirement.codeFilter.path[1]')}
+                    <h5>
+                      {codeFilterPath2[0].charAt(0).toUpperCase()}
+                      {codeFilterPath2[0].slice(1)}:
+                    </h5>
                   </Grid>
                   <Grid item xs>
                     {fhirpath.evaluate(response, 'dataRequirement.codeFilter.code.code').map((code: string) => {
                       return (
                         <Grid item xs key={code}>
-                          code: {code}
+                          - {code}
                         </Grid>
                       );
                     })}
                   </Grid>
                   <Grid item xs>
-                    <h5>dateFilter:</h5>
-                  </Grid>
-                  <Grid item xs>
-                    path: {fhirpath.evaluate(response, 'dataRequirement.dateFilter.path')}
+                    <h4>Dates:</h4>
+                    <h5>{fhirpath.evaluate(response, 'dataRequirement.dateFilter.path')}:</h5>
                   </Grid>
                   <Grid item xs>
-                    valuePeriod:
-                  </Grid>
-                  <Grid item xs>
-                    start: {fhirpath.evaluate(response, 'dataRequirement.dateFilter.valuePeriod.start')}
-                  </Grid>
-                  <Grid item xs>
-                    end: {fhirpath.evaluate(response, 'dataRequirement.dateFilter.valuePeriod.end')}
+                    {startDate.toDateString()}-{endDate.toDateString()}
                   </Grid>
                 </Grid>
-                <Grid container item xs={6} direction="column">
+                <Grid container xs direction="column">
                   <Grid item xs>
-                    <h4>reasonCode:</h4>
-                  </Grid>
-                  <Grid item xs>
-                    system: {fhirpath.evaluate(response, 'reasonCode.coding.system')}
-                  </Grid>
-                  <Grid item xs>
-                    code: {fhirpath.evaluate(response, 'reasonCode.coding.code')}
-                  </Grid>
-                  <Grid item xs>
-                    display: {fhirpath.evaluate(response, 'reasonCode.coding.display')}
+                    <h4>Reason(s):</h4>
+                    {fhirpath.evaluate(response, 'reasonCode').map((reason: R4.IGuidanceResponse) => {
+                      return (
+                        <Grid item xs key={fhirpath.evaluate(reason, 'coding.code')}>
+                          - {fhirpath.evaluate(reason, 'coding.code')} ({fhirpath.evaluate(reason, 'coding.display')})
+                        </Grid>
+                      );
```

Comment:
Let's pull the `coding` out to a variable with fhirpath and then access the properties directly rather than doing 3 evaluates inline